recipes/server_install_only.rb

```diff
@@ -0,0 +1,17 @@
+# Cookbook Name:: cerner_splunk
+# Recipe:: server_install_only
+#
+# Only run the basic install for the purposes of baking a base server image. Skips
+# the vast majority of configuration since it's likely environment specific.
+#
+
+fail 'Server installation not currently supported on windows' if platform_family?('windows')
+
+## Attributes
+instance_exec :server, &CernerSplunk::NODE_TYPE
+node.run_state['cerner_splunk'] ||= {}
```

Comment:
Very minor, and possibly bikeshedding... but if we put this in an attributes file, could we then just use it anywhere without the extra falsy checks?

Reply:
Seems to work, and reduces a few lines of code: https://github.com/cerner/cerner_splunk/pull/160/commits/6d076e50ee34a53b13a17c3007a7e083715893d9